test/utils.cpp

```diff
@@ -1981,3 +1982,275 @@ TEST_CASE("translit() always returns the same value for the same inputs",
 		}
 	}
 }
+
+void verify_convert_text(const std::vector<unsigned char>& input,
+	const std::string& tocode, const std::string& fromcode,
+	const std::vector<unsigned char>& expected_output)
+{
+	const std::string input_str = std::string(reinterpret_cast<const char*>(input.data()),
+			input.size());
+	const std::string expected_str = std::string(reinterpret_cast<const char*>
+			(expected_output.data()), expected_output.size());
+
+	REQUIRE(utils::convert_text(input_str, tocode, fromcode) == expected_str);
+}
+
+TEST_CASE("convert_text() returns input string if fromcode and tocode are literally the same",
+	"[Utils]")
```

Comment:
Please make the test tag lowercase. They reflect the name of the class or namespace under test, and since namespaces in Newsboat are all lowercase, the tag should be too.

Reply:
ah oops, I somehow misread the tag of the test above.
Fixed and rebased